spec/controllers/events_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe EventsController, type: :controller do
+  describe "#index" do
+    context "events page" do
+      it "lists the events, ordered by start asc in UTC" do
+        event1 = FactoryBot.create(:event)
+        event2 = FactoryBot.create(:event, start: "2019-09-08 03:00:00", finish: "2019-09-08 20:00:00", error: nil, success: true, created_at: "2019-09-08 20:00:00", updated_at: "2019-09-08 20:00:00")
+        event3 = FactoryBot.create(:event, start: "2019-11-07 03:00:00", finish: "2019-11-07 20:00:00", error: nil, success: true, created_at: "2019-11-07 20:00:00", updated_at: "2019-11-07 20:00:00")
+        event4 = FactoryBot.create(:event, start: "2019-12-07 03:00:00", finish: "2019-12-07 20:00:00", error: nil, success: true, created_at: "2019-12-07 20:00:00", updated_at: "2019-12-07 20:00:00")
+        get :index
+        expect(assigns(:events)).to eq([event1, event2, event3, event4])
```

Comment:
I made comments on the other test that I think apply here as well.

Reply:
This test will fail only if I change the order of the events in the index. If for example I change it to desc. It is difficult to test the database because I don't know if it is something that was created because of the pg_loader and how the migration happened from mysql to postgres. The important thing is to force the order of the events to be asc and this is what I'm testing.